src/hooks/useTransactionStatus.ts

```diff
@@ -20,6 +20,7 @@ const STATUS_LABELS: Record<TxLocalStatus, string> = {
   [PendingStatus.SUBMITTING]: 'Submitting',
   [PendingStatus.PROCESSING]: 'Processing',
   [PendingStatus.INDEXING]: 'Indexing',
+  [PendingStatus.SIGNING]: 'Signing',
```

Comment:
Question for reviewer: Is `Signing` a good label for this status?
Or should it be something like `Adding confirmation` or `Indexing confirmation`?

Reply:
I think it's a good name!